lwt/tls_lwt.ml

```diff
@@ -18,6 +18,8 @@ let resolve host service =
   | ai::_ -> return ai.ai_addr
 
 
+module Unix_ = Unix
+
```

Comment:
that's pretty ugly.  You only use this for `gettimeofday`, so why not replace this with `let gettimeofday = Unix.gettimeofday` ?